README.md

```diff
@@ -111,10 +160,16 @@ Note that `parameters` in both constructor and function call are mapped to sCryp
 
 * `boolean`: mapped to sCrypt `bool`
 * `number`: mapped to sCrypt `int`
-* `new Byte(x)` / `new Bytes(x)` / `new Sig(x)` / `new PubKey(x)` / `new Ripemd160(x)` / … : mapped to sCrypt `byte` / `bytes` / `Sig` / `PubKey` / `Ripemd160` / … , where `x` is hex string
+* `new Int(x)`/ `new Bool(x)` / `new Bytes(x)` / `new Sig(x)` / `new PubKey(x)` / `new Ripemd160(x)` / … : mapped to sCrypt `int` / `bool` / `bytes` / `Sig` / `PubKey` / `Ripemd160` / … , where `x` is hex string
 
 In this way, the type of parameters could be checked and potential bugs can be detected before running.
 
+Other complex types are the structure and type aliases of user positioning. They need to be dynamically generated and you can use `buildTypeClasses` to do that.
```

Comment:
Composite types, including structs and type aliases, are dynamically generated by `buildTypeClasses`.